lib/src/importer-registry.ts

```diff
@@ -41,12 +48,35 @@ export class ImporterRegistry<sync extends 'sync' | 'async'> {
       );
   }
 
+  // Type predicate for NodePackageImporter
+  isNodePackageImporter(
+    importer: Importer<sync> | FileImporter<sync> | NodePackageImporter
+  ): importer is NodePackageImporter {
+    return typeof importer === 'symbol';
```

Comment:
this is wrong. You should not accept _any_ symbol as being the NodePackageImporter.

The check should be `importer === nodePackageImporter`

Reply:
This is not the equality check- that happens within `register()`. This is (loosely) a type predicate, which allows us to differentiate between NodePackageImporter-like values and Importer/FileImporter-like values. 

But you're right this is unclear, and since we've simplified the type now, I can just move this check inside `register()`, which I did in https://github.com/sass/embedded-host-node/pull/260/commits/e591fadfa1911d887a81648e3b9b677703004c59. Thanks!